server/jwt/jwt_utils_test.go

```diff
@@ -0,0 +1,83 @@
+package jwt
+
+import (
+	"testing"
+
+	"github.com/fabiocicerchia/go-proxy-cache/config"
+	"github.com/fabiocicerchia/go-proxy-cache/utils/slice"
+)
+
+func TestContains(t *testing.T) {
+
+	v := []string{"a", "b"}
+	res := slice.ContainsString(v, "a")
+	if !res {
+		t.Error("Expected true but got", res)
+	}
```

Comment:
by using the library `github.com/stretchr/testify/assert` present already in other tests this would be more readable:

```suggestion
	res := slice.ContainsString(v, "a")
	assert.True(res)
```